src/pages/chainlink-nodes/contracts/receiver.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: ../../../layouts/MainLayout.astro
+section: nodeOperator
+date: Last Modified
+title: "Receiver"
+---
+
+[AuthorizedReceiver](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.7/AuthorizedReceiver.sol) is an abstract contract inherited by [Operator](/chainlink-nodes/contracts/operator) and [Forwarder](/chainlink-nodes/contracts/forwarder) contracts.
+
+## Api Reference
+
+### Methods
+
+#### setAuthorizedSenders
+
+```solidity
+function setAuthorizedSenders(address[] senders) external
+```
+
+Sets the fulfillment permission for a given node. Use `true` to allow, `false` to disallow.
```

Comment:
This section is not very clear to me what it means. Can you please clarify?

Reply:
based on https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.7/AuthorizedReceiver.sol#L13